printer_mods/Majarspeed/Hartk_Mount_Galileo/README.md

```diff
@@ -0,0 +1 @@
+Created folder
```

Comment:
Please describe the mod in this README, including a link to hartk's board.  Also include the image of the mount that you have in this PR.

Reply:
No, I started working on it and got busy and never came back that night to finish it. All the edits should be good now. Created a folder all the spaces should be gone added an image and wrote a very simple description with link to git for the board and a purchase link.